### ISSUE TITLE
🐛 [BUG] - Case-senstitivy issue in JS reference used by is_imageslider module

### ISSUE THREAD

**Issue by loremipsum99** [Oksydan/falcon#174]
### Description

Hi

As it's not possible to report an issue specifically for the is_imageslider module project I'd like to report my issue here.

I've found that the is_imageslider is not working properly in the BO on systems where the request-path is case-sensitive (such as in case of an Apache driven web-server on a Linux OS).

The problem is that the assets in views/templates/admin/form.html.twig and views/templates/admin/index.html.twig are referenced with a capital M for the module folder that is part of the path for the asset. In a default prestashop installation, the module folder is written in lower-case. As result, the referenced assets cannot be loaded on a system that is case-sensitive.

Example from line 43 from https://github.com/Oksydan/is_imageslider/blob/main/views/templates/admin/index.html.twig#L43:

`<script src="{{ asset('../Modules/is_imageslider/views/js/form.js') }}"></script>`

Fix:
Simply write /modules/ all lower-case. Example:

`<script src="{{ asset('../modules/is_imageslider/views/js/form.js') }}"></script>`

### Node.js version

v14.21.2

### php version

v8.1

### OS and it's version

Ubuntu 22.04

### Browsers

Firefox

### Required module/theme

is_imageslider

### Reproduction steps

```bash
1. Install starter theme on a case sensitive system (e.g. Apache running on Linux)
2. Navigate to the is_imageslider module settings in the BO
3. Attempt to add a new slider
4. Form looks broken -> attempting to select an image to be uploaded, no preview is displayed. In the browser console you can see that the referenced asset (createForm.js) cannot be found.
```

**Comment by Oksydan** [comment]
Hi @loremipsum99,

I was able to reproduce an issue.
I will fix it with new version 2.1.0.
You are not able to create issue for specific repository coz it is easier for me to track only on issue tracker for this project instead of multiple for each. 

Thank you for creating an issue.

**Comment by Oksydan** [comment]
Hi @loremipsum99,

new version has been released https://github.com/Oksydan/is_imageslider/releases/tag/v2.1.0 
I am closing this issue. 
Thank you for creating an issue.
